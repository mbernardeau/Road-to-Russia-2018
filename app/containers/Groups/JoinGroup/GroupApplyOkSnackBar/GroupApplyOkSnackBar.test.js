import React from 'react'
import renderer from 'react-test-renderer'

import GroupApplyOkSnackbar from './GroupApplyOkSnackBar'

xdescribe('containers/JoinGroup/GroupApplyOkSnackbar', () => {
  it('renders correctly', () => {
    const props = {
      open: true,
      handleRequestClose: () => {},
    }

    const tree = renderer.create(<GroupApplyOkSnackbar {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
