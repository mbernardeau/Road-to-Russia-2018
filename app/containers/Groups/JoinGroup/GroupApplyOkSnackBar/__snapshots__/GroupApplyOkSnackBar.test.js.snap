// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/JoinGroup/GroupApplyOkSnackbar renders correctly 1`] = `
<div
  className="MuiSnackbar-root-1 MuiSnackbar-anchorBottomLeft-7"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    aria-describedby="message-id"
    className="MuiTypography-root-38 MuiTypography-body1-47 MuiPaper-root-11 MuiPaper-shadow6-19 MuiSnackbarContent-root-8"
    direction="up"
    role="alertdialog"
  >
    <div
      className="MuiSnackbarContent-message-9"
    >
      <span
        id="message-id"
      >
        Demande envoy√©e
      </span>
    </div>
    <div
      className="MuiSnackbarContent-action-10"
    >
      <button
        aria-label="Close"
        className="MuiButtonBase-root-71 MuiIconButton-root-62 MuiIconButton-colorInherit-66"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label-68"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-73 MuiIconButton-icon-69"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-74"
        />
      </button>
    </div>
  </div>
</div>
`;
